arbitrum-docs/node-running/how-tos/data-availability-committee/partials/_2.4-das-configuration-options-state-sync.mdx

```diff
@@ -0,0 +1 @@
+DA servers also have an optional REST aggregator which, when a data batch is not found in cache or storage, requests that batch to other REST servers defined in a list and stores that batch upon receiving it. This is how a DAS that misses storing a batch (the AnyTrust protocol doesn't require all of them to report success in order to post the batch's certificate to the parent chain) can automatically repair gaps in the data it stores, and also how a mirror DAS can sync its data. A public list of REST endpoints is published online, which the DAS can be configured to download and use, and additional endpoints can be specified in the configuration.
```

Comment:
Flag - this partial can be used above (seeing duplication of this paragraph in `deploy-a-das`)